central/complianceoperator/v2/checkresults/datastore/datastore_impl.go

```diff
@@ -281,6 +283,20 @@ func (d *datastoreImpl) CountByField(ctx context.Context, query *v1.Query, field
 	return 0, errors.Errorf("Unable to group result counts by %q", field)
 }
 
+func (d *datastoreImpl) WalkByQuery(ctx context.Context, query *v1.Query, fn func(deployment *storage.ComplianceOperatorCheckResultV2) error) error {
+	wrappedFn := func(checkRes *storage.ComplianceOperatorCheckResultV2) error {
+		d.updateCheckResultPriority()
+		return fn(checkRes)
+	}
+	return d.store.WalkByQuery(ctx, query, wrappedFn)
+}
+
+func (d *datastoreImpl) updateCheckResultPriority(checkResults ...*storage.ComplianceOperatorCheckResultV2) {
+	for _, checkResult := range checkResults {
+		checkResult.Priority = d.checkResultRanker.GetRankForID(checkResult.GetId())
+	}
+}
```

Comment:
The ranker stuff is for things with CVSS scores and such.  It does not apply to compliance.